spring-session/src/test/java/org/springframework/session/jdbc/config/annotation/web/http/JdbcHttpSessionConfigurationTests.java

```diff
@@ -181,6 +207,17 @@ public PlatformTransactionManager transactionManager() {
 	}
 
 	@Configuration
+	static class CustomTableNameSetConfiguration extends JdbcHttpSessionConfiguration {
+
+		@Autowired
+		public void customize() {
```

Comment:
Why a random ```@Autowired``` method?
Simply overriding the default constructor and using the desired setter inside it is much cleaner and more understandable.

Also separate config for each test are preferred.